src/licenses/licenses.xml

```diff
@@ -649,7 +649,7 @@
       </licenses>
     </dependency>
     <dependency>
-      <groupId>io.openshift.boosters</groupId>
+      <groupId>io.openshift.booster</groupId>
```

Comment:
What is this groupId about ? (As we are dropping the 'booster')

Reply:
Licenses will be regenerated during booster release. I just automatically replaced `io.openshift.boosters` by `io.openshift.booster` before renaming entirely.